src/main/interface/settings.c

```diff
@@ -860,6 +862,12 @@ const clivalue_t valueTable[] = {
 #ifdef USE_RANGEFINDER
     { "rangefinder_hardware", VAR_UINT8 | MASTER_VALUE | MODE_LOOKUP, .config.lookup = { TABLE_RANGEFINDER_HARDWARE }, PG_RANGEFINDER_CONFIG, offsetof(rangefinderConfig_t, rangefinder_hardware) },
 #endif
+
+#ifdef USE_WATCHDOGS
+    { "wds_targets", VAR_UINT8 | MASTER_VALUE , .config.minmax = { 0, 3 }, PG_WATCHDOGS_CONFIG, offsetof(watchdogsConfig_t, enabled_watchdogs) },
```

Comment:
This should be done cleaner. See `rc_interp` for how to do this with a lookup table.

Reply:
@mikeller Thanks for your comment, will take a look and follow the same!

Reply:
@mikeller , if I read the code correctly, it appears that lookup table supports only single value selection. I want to be able set the value to roll, or pitch, or both (roll | pitch). I can probably add new value 'rollandpitch' to the enum to achieve the same result, but this will not scale if we have more than two options. Is that what you have in mind?